test/unit/helpers/xhr/PASSWORD_RESET_withComplexity.js

```diff
@@ -7,6 +7,10 @@ define({
     "status": "PASSWORD_RESET",
     "_embedded": {
       "policy": {
+        "age": {
+          "minAgeMinutes": -1,
+          "historyCount": 0
```

Comment:
If I will keep here non-zero values I will have to reset them all for testing only one complexity rule.
Do you think it is a better way?